packages/expo-contacts/src/__stories__/components/AddContact.tsx

```diff
@@ -0,0 +1,26 @@
+import * as Contacts from 'expo-contacts';
+import { Button } from 'expo-stories/shared/components';
+import * as React from 'react';
+
+import { useContact } from '../helpers';
+import { ContactCard } from './ContactCard';
+
+type IAddContact = {
```

Comment:
I'm not a big fun of these `I<TypeName>` prefixes 🤔 Why not to follow the idea of `AddContactProps` (mimicking the RN's types?)

Reply:
this is an old habit of mine - sorry!